pkg/apis/kubeflow.org/v1/paddlepaddle_defautls.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2018 The Kubeflow Authors
```

Comment:
Probably, this is `// Copyright 2022 The Kubeflow Authors`, not `// Copyright 2018 The Kubeflow Authors`.

Reply:
done.